client/controllers/events/eventFiltration.coffee

```diff
@@ -228,8 +241,24 @@ Template.eventFiltration.events
   'click .locations .deselect-all': (event, instance) ->
     instance.selectedLocations.remove({})
 
-  'change .dates input': (event, instance) ->
-    dates = []
-    instance.$('.dates input').each (i, input) ->
-      dates.push(new Date(input.value))
-    instance.selectedDateRange.set(dates)
+  'dp.change': (event, instance) ->
+    startStr = $('.start-date').data('DateTimePicker').date()?.utc().format('YYYY-MM-DD')
```

Comment:
The date time picker displays locale dates. Interpreting them as utc could cause them to have different values that what the user entered.